src/main/java/competition/commandgroups/MoveCollectedGamepieceToArmCommandGroup.java

```diff
@@ -58,10 +59,15 @@ public MoveCollectedGamepieceToArmCommandGroup(
                 () -> UnifiedArmSubsystem.KeyArmPosition.PrepareToAcquireFromCollector,
                 () -> UnifiedArmSubsystem.RobotFacing.Forward);
 
-        var waitThenPullArm = new WaitCommand(0.1).andThen(cubePullOutGamePieceCommand);
+        //Instead of waitThenPullArm, will use the clawConeIntake with a small change
+        //var waitThenPullArm = new WaitCommand(0.1).andThen(cubePullOutGamePieceCommand);
+        var clawCubeIntake = clawMotors.createIntakeCommand()
+                .withTimeout(cubeClawIntakeTime)
+                .andThen(clawMotors.createStopCommand().withTimeout(0.01));
+
         var cubePullSequence = collector.getEjectThenStopCommand()
                 .withTimeout(cubeCollectorEjectTime)
-                .alongWith(waitThenPullArm);
+        //      .alongWith(waitThenPullArm); Would this be neccesary if the cube is secured in claw?
```

Comment:
if not the collector might keep trying to pull the cube back?

Reply:
Ahh alright, adding that back